example_project/example_project_combined.json

```diff
@@ -838,8 +840,8 @@
         "name": "Residential 5",
         "type": "Building",
         "building_type": "Multifamily",
-        "floor_area": 28636,
-        "footprint_area": 14318,
+        "floor_area": 10000,
+        "footprint_area": 5000,
```

Comment:
I need to reduce these because this feature was resulting in no matches (each units' floor area was too large).